examples/typescript/batch_mint.ts

```diff
@@ -50,7 +50,7 @@ async function main() {
   // First need to create a collection on chain
   console.log("creating collection...");
   const transaction = await aptos.createCollectionTransaction({
-    creator: sender,
+    creator: sender.accountAddress,
```

Comment:
So I would suggest instead of this, add Account to the AccountAddressInput type.

Then update the from function in accountAddress.ts

static from(input: AccountAddressInput): AccountAddress {
    if (input instanceof Account) {
      return input.accountAddress;
    }
    if (input instanceof AccountAddress) {
      return input;
    }
    if (input instanceof Uint8Array) {
      return new AccountAddress(input);
    }
    return AccountAddress.fromString(input);
  }

Reply:
I like it

Reply:
Using `instanceof` requires importing the full type, and causes a cylical dependency :(

Reply:
I'm doing this instead, should still work:
```typescript
  static from(input: AccountAddressInput): AccountAddress {
    if (input instanceof AccountAddress) {
      return input;
    }
    if (input instanceof Uint8Array) {
      return new AccountAddress(input);
    }
    if (typeof input === "string") {
      return AccountAddress.fromString(input);
    }
    return input.accountAddress;
  }
```

Reply:
Adding the `Account` requires changing the code in a lot of places specifically before generating an entry function payload where we're passing an `AccountAddressInput`.

Both `AccountAddress` and `string` are valid entry function payload argument inputs, but if we add `Account` it starts complaining.
To solve that we would either have to:
- Use `AccountAddress.from` every time we're passing an `AccountAddressInput` to the function arguments
  I don't think we should go down this path as it adds more complexity especially for the end user.

- Add support for `Account` in the layer that takes care of generating / submitting an entry function payload.
  This could be a good option, but I'm not familiar with that part of the code and worry about the time needed. Happy to give it a shot later.

Reply:
Ok yeah, kinda annoying.  Don't worry about it within the scope of this PR.

Do note, this does still constitute a breaking change, but its trivial.